tests/foreman/rhai/test_rhai_client.py

```diff
@@ -69,21 +73,106 @@ def setUpClass(cls):  # noqa
         cls.org_name = org.name
 
     def test_positive_connection_option(self):
-        """Verify that '--test-connection' option for redhat-access-insights
-        client rpm tests the connection with the satellite server connection
-        with satellite server
+        """Verify that '--test-connection' option for insights-client rpm tests
+        the connection with the satellite server connection with satellite
+        server
 
         :id: 167758c9-cbfa-4a81-9a11-27f88aaf9118
 
-        :expectedresults: 'redhat-access-insights --test-connection' should
+        :expectedresults: 'insights-client --test-connection' should
             return zero on a successfully registered machine to RHAI service
         """
         with VirtualMachine(distro=DISTRO_RHEL6) as vm:
             vm.configure_rhai_client(self.ak_name, self.org_label,
                                      DISTRO_RHEL6)
             test_connection = vm.run(
-                'redhat-access-insights --test-connection')
+                'insights-client --test-connection')
             self.logger.info('Return code for --test-connection {0}'.format(
                 test_connection.return_code))
             self.assertEqual(test_connection.return_code, 0,
                              '--test-connection check was not successful')
+
+    def test_provision(self):
+        """A new host appears in Red Hat Access Insights inventory
+
+        :id: 0bf2bccf-0973-49fa-923f-7409fba86f85
+
+        :expectedresults: provisioned host is in the inventory
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.configure_rhai_client(
+                activation_key=self.ak_name,
+                org=self.org_label,
+                rhel_distro=DISTRO_RHEL7)
+            self.assertIn(
+                'Client: 3.',
+                vm.run('insights-client --version').stdout[0],
+            )
+            with Session(self) as session:
+                set_context(session, org=self.org_label)
+                vm_info = session.hosts.get_host_properties(
+                    vm.hostname,
+                    ['Status', 'Operating system', 'Subscription'])
+                self.assertIn(vm_info['Status'], ['Warning', 'OK'])
```

Comment:
a little bit ambiguous =)

Reply:
I guess status  it's ok. When a host is provisioned it can have either in Warning or OK status.